ex2/ex2.c

```diff
@@ -8,7 +8,17 @@
 
 int main(void)
 {
-    // Your code here 
-    
+    FILE *file = fopen("text.txt", "r+");
+    int rv = fork();
+    if (rv == 0)
+    {
+        fprintf(file, "%s\n", "child");
+        printf("FILE-C: %p\n", file);
+    }
+    else
+    {
+        fprintf(file, "%s\n", "parent");
+        printf("FILE-P: %p\n", file);
+    }
     return 0;
```

Comment:
This is interesting! It prints out the pointers to the parent and child files instead of just a character string. Nice job!